docs/backend-services.md

```diff
@@ -0,0 +1,48 @@
+# Backend dependencies and architecture
+
+The application is built as a single page web app that can run without its own dedicated backend. This is useful for
+hosting it with a very simple, static host, like github pages. The server serves the static web content and the frontend
+calls the [JW player platform Delivery API](https://developer.jwplayer.com/jwplayer/docs) directly.
```

Comment:
Small nitpick
```suggestion
calls the [JW Player Platform Delivery API](https://developer.jwplayer.com/jwplayer/docs) directly.
```

Reply:
```suggestion
calls the [JW Player Delivery API](https://developer.jwplayer.com/jwplayer/docs) directly.
```

Reply:
The Management API is a the  _Platform_ Management API. The other API is merely Delivery API.